packages/utah-design-system/src/components/LayerSelector.stories.tsx

```diff
@@ -77,6 +78,18 @@ export function Default({
       options: {
         view,
         quadWord: import.meta.env.VITE_QUAD_WORD,
+        basemaps: [
+          {
+            label: 'Vector Lite',
+            function: () =>
+              new Basemap({
+                portalItem: {
+                  id: '98104c602b7c44419c0a952f28c65815',
+                },
+              }),
+          },
+          'Lite', // this is for testing the happy path tokens
+        ],
```

Comment:
These two items are functionally equivalent correct? It is slightly confusing using the same token for a portal item and discover. I assume you thought about this and have reasons that it's wise to keep them the same?

Reply:
Yes, they are the same. I probably should use a different map for the token example.

I agree about using a vector-specific token. However, the token is also the label and I didn't think that we wanted or needed the word "Vector" to show up in the radio button label.